src/hb-ot-shape-complex-indic.cc

```diff
@@ -1006,11 +1006,14 @@ initial_reordering_indic (const hb_ot_shape_plan_t *plan,
 			  hb_font_t *font,
 			  hb_buffer_t *buffer)
 {
+  if (!buffer->message (font, "start reordering indic initial"))
```

Comment:
How about using hyphens or underscores in “indic initial” and “indic final”, so splitting on spaces returns the three pieces of the “action item-type identifier” format?

Reply:
Hmm. I can see arguments both ways. :-) I was thinking of initial/final as another sub-level in the information hierarchy.